src/io/tradetrust/certificate-of-origin/1.0/schema.test.ts

```diff
@@ -1,172 +1,76 @@
-import Ajv from "ajv";
-import schema from "./schema.json";
-import sampleDocument from "./certificate-of-origin-wrapped.json";
-import { cloneDeep, omit } from "lodash";
+import CertificateOfOriginSchema from "./CertificateOfOrigin.v3.json";
```

Comment:
name is weird, it makes it sounds it's a schema but it's not

Reply:
okay, changed to import context.